sdk/entra/WebJobs.Extensions.AuthenticationEvents/test/WebJobs.Extensions.AuthenticationEvents.Tests.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
This should be using `$(RequiredTargetFrameworks)`; we target a variety of frameworks for testing.

Reply:
I tried this but I get this: (Which is odd as it works on my other project)
![image](https://user-images.githubusercontent.com/89389217/184403589-85901b86-0ff9-4c99-ae59-5edc3d453a38.png)

Reply:
Try adding a `Directory.Build.props` ?

Reply:
It will change to using the standard repository key.   You'll want to update your `InternalsVisibleTo` to the following:

`PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4`

- [Example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/src/Properties/AssemblyInfo.cs#L7)

Reply:
I've tried numerous variations, I still cannot get this to work. Any other suggestions?

Reply:
I think I found the issue... once I renamed the test lib to be exactly the same as the main lib suffixed with '.tests' it seem to work...

Reply:
Hmm, do you need to push the changes?

Reply:
Yes, sorry, I'm about to now, just running through the new comments and then will push.